pkg/trace/traceutil/azure.go

```diff
@@ -37,7 +37,7 @@ const (
 	appService = "app"
 )
 
-var appServicesTags map[string]string
+var appServicesTags map[string]string //TODO: make this actually cache?
```

Comment:
I'd rather remove the variable and move the `// TODO: add caching` in `GetAppServicesTags`